packages/authentication/modules/user/useUpdateUser/index.ts

```diff
@@ -23,6 +25,7 @@ const useUpdateUser = <TUser extends Pick<User, 'id'>>({
       } catch (e) {
         // silently fail
         // eslint-disable-next-line no-console
+        setCurrentProfile(null)
```

Comment:
The reason I put this here, is that if there occurs an error in `refreshAccessToken` this function removes the access and refresh tokens, but does not fire a `logout` event. So I guess we should also remove the current profile, too. I would have liked to do that in the error handling of the `refreshAccessToken` function directly, but since the function is defined in the `utils` package, it should better not import from `authentication`. I thought that passing an onError callback was too compicated, so settled for this easier solution.